scripts/install.sh

```diff
@@ -30,15 +30,15 @@ if [[ $editor == "NO EDITOR" ]]; then
 fi
 
 echo "Choosen editor: $editor. You can change it as wrote in README"
-sed -i "/djvuviewer_script/s/=.*\.sh/=~\/gnu-koryavov\/$editor.sh/" ../config.conf
+sed -i "/djvuviewer_script/s/=.*\.sh/=~\/gnu-koryavov\/$editor.sh/" $main_config
 
 
 echo $HOME
 
 if [[ $ans == "y"* || $ans == "Y"*  ]]; then
 
     mkdir -p -v $HOME/$home_dir/KORYAVNIKS
-    cp ../config.conf $HOME/$home_dir
+    cp ./../configs/config.conf $HOME/$home_dir
```

Comment:
Why is it better than the previous one?

Reply:
In new version it is variable with post-install config path for more usable editing of config in install

Reply:
I mean why do we use `./../configs/` instead of `..configs/`?

Reply:
I changed it, but difference is very small (may be 2 octets in file size and faster directory tree processing)